cvat-ui/src/utils/opencv-wrapper/opencv-wrapper.ts

```diff
@@ -73,13 +77,8 @@ export class OpenCVWrapper {
         OpenCVConstructor();
 
         const global = window as any;
-        await waitFor(
-            100,
-            () =>
-                typeof global.cv !== 'undefined' && typeof global.cv.segmentation_IntelligentScissorsMB !== 'undefined',
-        );
 
-        this.cv = global.cv;
+        this.cv = await global.cv;
```

Comment:
Does ``global.cv`` return Promise? global variable used in waitFor, but now you probably do not need it.

Reply:
Yes, it returns Promise now, but i dont think we can get rid of it because this promise is stored in global variable anyway